internal/controller/groupqualityofservicelimits/groupqualityofservicelimits.go

```diff
@@ -142,8 +147,29 @@ func (c *external) Observe(ctx context.Context, mg resource.Managed) (managed.Ex
 		return managed.ExternalObservation{}, errors.New(errNotGroupQualityOfServiceLimits)
 	}
 
-	// These fmt statements should be removed in the real implementation.
-	fmt.Printf("Observing: %+v", cr)
+	group := cr.Spec.ForProvider.GroupID
+	if group == "" {
+		return managed.ExternalObservation{}, nil
+	}
+
+	user := cloudian.User{
+		GroupID: cr.Spec.ForProvider.GroupID,
```

Comment:
```suggestion
		GroupID: group,
```

Reply:
Do you mean

```go
	group := cloudian.User{
		GroupID: cr.Spec.ForProvider.GroupID,
		UserID:  "*",
	}
```

Reply:
Aah, outside of the diff view :D See it now!